[ja_JP] Translate "access_control.md" into Japanese (#706)
hyperledger/fabric-docs-i18n: docs/locale/ja_JP/source/access_control.md

```

> このポリシー構造体は次のように解釈できます。*`MyPolicy` という名前のポリシーは、"Org1のpeer"または"Org2のpeer"の役割を持つアイデンティティの署名によってのみ満たされます*。

Reviewer: ```suggestion
このポリシー構造体は次のように解釈できます。*`MyPolicy` という名前のポリシーは、"Org1のピア"または"Org2のピア"の役割を持つアイデンティティの署名によってのみ満たされます*。
```
クォートの中身は説明だと思いますので、用語集に従ってピアとしたほうがよいと思います。

> Signatureポリシーは、 `AND` 、 `OR` 、 `NOutOf` の任意の組み合わせをサポートしており、"An admin of org A and two another admins or 11 of 20 org admins"のように非常に強力なルールを記述することもできます。

Reviewer: ```suggestion
Signatureポリシーは、 `AND` 、 `OR` 、 `NOutOf` の任意の組み合わせをサポートしており、"組織Aの管理者とそれ以外の2組織の管理者、もしくは、20組織のうち11組織の管理者"のように非常に強力なルールを記述することもできます。
```
Policyとして有効な文ではないので説明と解釈して、他と同様に中身を訳したほうがよいでしょう。

    Rule: "MAJORITY Admins"
```

ここで、 `AnotherPolicy` ポリシーは、 `MAJORITY` の `Admins` により満たされます。
`Admins` は、より低レイヤの `Signature` ポリシーで定義されます。

### Where is access control specified?

アクセス制御のデフォルトは `configtx.yaml` に存在します。
`configtxgen` は、このファイルを利用してチャネル設定をビルドします。

アクセス制御は、次のいずれかの方法で更新できます。
1つ目は、 `configtx.yaml` を編集し、新しいチャネル設定を生成する時に利用する方法です。
2つ目は、既存チャネルのチャネル設定内のアクセス制御を更新する方法です。

## How ACLs are formatted in `configtx.yaml`

> ACLは、リソース関数名の後にストリングが続くキーバリューペアで記述されます。

Reviewer: ```suggestion
ACLは、リソース関数名の後に文字列が続くキーバリューペアで記述されます。
```
用語集にはないですが、カタカナはあまり一般的ではないかと思います。
